diff --git a/net/interfaces/interfaces.go b/net/interfaces/interfaces.go
index 216521bfa..1926b9c88 100644
--- a/net/interfaces/interfaces.go
+++ b/net/interfaces/interfaces.go
@@ -35,7 +35,11 @@ func isProblematicInterface(nif *net.Interface) bool {
        // DoS each other by doing traffic amplification, both of them
        // preferring/trying to use each other for transport. See:
        // https://github.com/tailscale/tailscale/issues/1208
-       if strings.HasPrefix(name, "zt") || (runtime.GOOS == "windows" && strings.Contains(name, "ZeroTier")) {
+       if strings.HasPrefix(name, "zt") ||
+          strings.HasPrefix(name, "ppp") ||
+          strings.HasPrefix(name, "wg") ||
+          strings.HasPrefix(name, "tun") ||
+          (runtime.GOOS == "windows" && strings.Contains(name, "ZeroTier")) {
                return true
        }
        return false
